src/EditorFeatures/Core/Implementation/NavigateTo/NavigateToItemProvider.Searcher.cs

```diff
@@ -68,6 +68,9 @@ private class Searcher
                         await Task.WhenAll(searchTasks).ConfigureAwait(false);
                     }
                 }
+                catch (OperationCanceledException)
```

Comment:
It's basically an event handler.  The API between us and VS is that they simply call 'search', and that search is supposed to kick off work and then callback when it's done.  Hence 'void'.  however, we want to be able to use await's in our code so we don't have to kick off work and then do annoying .ContinueWiths.  Hence 'async'.  so 'async void' it is :-/